cpp/src/arrow/types/primitive.h

```diff
@@ -176,9 +184,10 @@ class PrimitiveBuilder : public ArrayBuilder {
   void AppendNulls(const uint8_t* null_bytes, int32_t length) {
```

Comment:
change null_bytes here to valid_bytes?  update comments above and below method signature?